apps/email/js/cards/compose.js

```diff
@@ -382,12 +394,8 @@ return [
     _saveNeeded: function() {
       var self = this;
       var checkAddressEmpty = function() {
-        var bubbles = self.querySelectorAll('.cmp-peep-bubble');
-        if (bubbles.length === 0 && !self.toNode.value && !self.ccNode.value &&
-            !self.bccNode.value) {
+        if (self.haveAddressesChanged()) {
```

Comment:
It is now fine to remove the `checkAddressEmpty` function here and just directly call `this.haveAddressesChanged()` in the final return check for `_saveNeeded`.